pkg/healthautoexport/types.go

```diff
@@ -61,6 +78,66 @@ type WorkoutFields map[string]*QtyWithUnit
 // https://stackoverflow.com/a/43178272/2037090
 type workoutCopy Workout
 
+func (m *Metric) UnmarshalJSON(bytes []byte) error {
+	type MetricInternal struct {
```

Comment:
If we move json.RawMessage into `Metric` itself, we don't need this anonymous type declaration here anymore as well